docs/requirements-docs.txt

```diff
@@ -1,6 +1,8 @@
 black  # required by mkdocstrings_handlers
 jinja2
 duckdb
+# Hopefully temporary until https://github.com/mkdocstrings/mkdocstrings/issues/716
+git+https://github.com/MarcoGorelli/griffe.git@no-overloads
```

Comment:
temporary (hopefully 🤞 ) workaround for https://github.com/mkdocstrings/mkdocstrings/issues/716